stack.yaml

```diff
@@ -1 +1,4 @@
 resolver: lts-12.0
+extra-deps:
+- time-1.9.1
+allow-newer: true
```

Comment:
Not sure that this is the right thing to do, but this line is needed to run the tests.

Without it, I get `Plan construction failed` when I try to run `stack test`.